<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemWindowStudent</class>
 <widget class="QMainWindow" name="SystemWindowStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>961</width>
      <height>611</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Schedule</string>
     </attribute>
     <widget class="QTableWidget" name="tableSchedule">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>941</width>
        <height>561</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>14</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>37</number>
      </attribute>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>19</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>21</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Monday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thursday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Friday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Saturday</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sunday</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>My Sections</string>
     </attribute>
     <widget class="QTableWidget" name="tableMyCourses">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>941</width>
        <height>531</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>Course Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Course Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Credits</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Section</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Room</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Days</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Capacity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Actual</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Courses</string>
     </attribute>
     <widget class="QTableWidget" name="tableCourses">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>921</width>
        <height>471</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Credits</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Sections</string>
     </attribute>
     <widget class="QTableWidget" name="tableSections">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>941</width>
        <height>431</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>Section Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Professor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Room</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Days</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Capacity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Actual</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>86</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>List of Courses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbCoursesList"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Course Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCourseName">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>641</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMessage">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>200</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Messages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbChangePassword">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>560</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogout">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>720</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRefresh">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>720</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbLog">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>740</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="openLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
